### PR TITLE
Add token's type to response of tokenlist

### PR DESCRIPTION
**Pull request by saneery**
https://github.com/poanetwork/blockscout/issues/1075

Added `token type` field to the `tokenlist` method

### CODE REVIEW

File: apps/block_scout_web/lib/block_scout_web/views/api/rpc/address_view.ex [hunk 1]
```diff
@@ -168,7 +168,8 @@ defmodule BlockScoutWeb.API.RPC.AddressView do
       "contractAddress" => to_string(token.contract_address_hash),
       "name" => token.name,
       "decimals" => to_string(token.decimals),
-      "symbol" => token.symbol
+      "symbol" => token.symbol,
+      "type" => token.type
```
Comment (ayrat555) [inline apps/block_scout_web/lib/block_scout_web/views/api/rpc/address_view.ex]: can you please add tests for `ERC-20` and `ERC-721` values in response?

Comment (saneery) [reply apps/block_scout_web/lib/block_scout_web/views/api/rpc/address_view.ex]: There are tests in PR. Maybe I don't understand what do you mean by those types, what the difference from other tests?

Comment (vbaranov) [reply apps/block_scout_web/lib/block_scout_web/views/api/rpc/address_view.ex]: @saneery I suppose @ayrat555 is talking about that there is only one token type passed through the tests of this endpoint `account_tokenlist_example_value`. It is of ERC-20 type. Let's add another test token of ERC-721 type too and pss it through the same test.

Comment (saneery) [reply apps/block_scout_web/lib/block_scout_web/views/api/rpc/address_view.ex]: @vbaranov I've added a new test token to `account_tokenlist_example_value`

**Pull request merged**